patches/netlify-lambda+2.0.16.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/node_modules/netlify-lambda/lib/build.js b/node_modules/netlify-lambda/lib/build.js
+index 4397356..e07c889 100644
```

Comment:
We should be able to get rid of `netlify-lambda` in general, swapping it out for a middleware in dev and shipping raw functions in prod, but that can be addressed later.